adyen.php

```diff
@@ -87,10 +87,15 @@ class Adyen extends PaymentModule
      */
     private $configuration;
 
+    /**
+     * @var Adyen\PrestaShop\model\AdyenPaymentResponse
+     */
+    private $adyenPaymentResponseModel;
+
     /**
      * Adyen constructor.
      *
-     * @throws \PrestaShop\PrestaShop\Adapter\CoreException
+     * @throws Adapter_Exception
```

Comment:
This is the exception for 1.6 you can find it in /Adapter/Adapter_Exception.php
I usually work on 1.6 that's why the auto phpdoc changed to this - it would be a good idea to stick to one version (1.7 I would say)

Reply:
For annotations I would stick to 1.7 as well.